instance/paneldb_config.cfg

```diff
@@ -6,3 +6,8 @@ SECRET_KEY='Ytvhsuasbnjj255ahsajakkaa1'
 
 #paneldb database connection URI:
 MONGODB_URI = 'mongodb://localhost:27017/paneldb'
+
+# Upload folder local
+UPLOAD_FOLDER='*'
+ALLOWED_EXTENSIONS = set(['txt', 'bed'])
```

Comment:
You can delete this line since this is handled by the HTML code <input type="file" accept=".bed, .txt, .csv" in the baitsets template file.

You could actually add to the accepted format also the excel (xls and xlsx)

Reply:
Done